sequencer/Models.py

```diff
@@ -4,7 +4,7 @@ class EnergyMaximizeReturn(Sequencer):
     """This class sequences a proposed electrification plan, optimizing for maximum Demand (kwh) / Distance (m)"""
     
     def nodal_demand(self, df):
-        return df['Demand...Projected.nodal.demand.per.year']
+        return df['peak.demand.in.kw']
```

Comment:
Note:  This change breaks backward compatibility.  May need to maintain v0.1.0 until migration to v0.2.0 is complete